### ISSUE TITLE
[Suggestion] Anti-Greenwashing Statement/Policy/Education

### ISSUE THREAD

**Issue by mkorbi** [cncf/tag-env-sustainability#46]
#### Description

Discussion originally started by @scottrigby [here](https://github.com/cncf/tag-env-sustainability/discussions/30) (was decided to move/use issues now). 

> discussion from meeting today:
it seems everyone agrees we do not want to do this, deliberately or accidentally
how to clearly send this message from the wg/tag
[Scott & Cara] Craft what our intentions are to make sure we are not greenwashing - create PR on existing Charter
add to glossary?
[Leo] how to avoid "policing"
Any other thoughts on this?

How we can clearly point out and educate about greenwashing and how to avoid it.

#### Impact

NA

#### Scope

NA

#### Additional info and comments

The initial statement against greenwashing was dropped out of charter, as it is not part of such.

**Comment by mkorbi** [comment]
yes, as stated it was initially within the charter -> no traffic.
But it didn't end up in the charter and is therefore still open. So we have to address it again or decide to close it.

**Comment by leonardpahlke** [comment]
We could bring it up in the TAG meeting and if someone would like to work on it, great and if not that's fine too and we close it.
